nifpga/bitfile.py

```diff
@@ -116,6 +116,8 @@ def _parse_type(type_xml):
         return _Array(name, type_xml)
     if type_name == "SGL" or type_name == "DBL":
         return _Float(name, type_name)
+    if type_name == "String":
+        return _String(name)
```

Comment:
Put a comment here that strings are not officially supported, but exist in error clusters